kubectl-hlf/cmd/ca/create.go

```diff
@@ -300,6 +311,7 @@ func newCreateCACmd(out io.Writer, errOut io.Writer) *cobra.Command {
 	f.StringArrayVarP(&c.caOpts.Hosts, "hosts", "", []string{}, "Hosts for Istio")
 	f.StringVarP(&c.caOpts.IngressGateway, "istio-ingressgateway", "", "ingressgateway", "Istio ingress gateway name")
 	f.IntVarP(&c.caOpts.IngressPort, "istio-port", "", 443, "Istio ingress port")
+	f.StringArrayVarP(&c.caOpts.ImagePullSecrets, "image-pull-secrets", "s", []string{}, "Image Pull Secrets for the CA Image")
```

Comment:
`-s` is already used for storage class option, can you consider using different, variable for pull secret.